annotto-front/src/modules/project/components/common/AnnotationItemWrapper.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import projectTypes from 'shared/enums/projectTypes'
+import { NER, TEXT, ZONE } from 'shared/enums/annotationTypes'
+
+import { findAnnotationItemType } from 'modules/project/services/annotationServices'
+
+import NerContainer from 'modules/project/components/common/NerContainer'
+import TextItemContainer from 'modules/project/components/common/TextItemContainer'
+import ImageMarker from 'modules/project/components/common/ImageMarker'
+
+const AnnotationItemWrapper = ({ projectType, tasks, currentItem, options }) => {
+  const annotationType = findAnnotationItemType(projectType, tasks)
+  const { body, highlights, data } = currentItem
+
+  switch (annotationType) {
+    case TEXT: {
+      return <TextItemContainer content={body} highlights={highlights} />
+    }
+
+    case NER: {
+      const filteredTasks = tasks.filter((task) => task.type === NER)
+      return <NerContainer content={body} {...options} highlights={highlights} tasks={filteredTasks} />
```

Comment:
I spread the props to avoid re-declaring them all. The purpose of the component is only to return the correct Annotation Component depending on the type of project and the associated tasks. It may be easier to read if they are all re-declared, but that will make the code heavier, right?